Vizzuality/half-earth-v3
# Fix/back button

## Fix Back button on AOI
### Description
Back button was not working correctly now it always redirects to data globe page
### Testing instructions
Go to any AOI or precalculated area. Click on layers. Click on the close button. It should redirect to the data globe
### Feature relevant tickets
https://vizzuality.atlassian.net/browse/HE-427?atlOrigin=eyJpIjoiNDUzYzhkMDVjYjIzNDZlZWI0Y2NlNjI5Yjc2M2YwYWIiLCJwIjoiaiJ9
